hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/SchedulerExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.mesos.hdfs.scheduler;
+
```

Comment:
I see that we don't have a choice here if we are to add a handler to properly log runtime exceptions. This makes sense in that case. Yes, I do suggest creating it as an inner class since its only a few lines of code.  Also, I noticed something else the Log is referencing ConfigServer.class which is not correct.